api-samples/tabs/inspector/tabs_api.js

```diff
@@ -0,0 +1,403 @@
+let tabs = {};
+let tabIds = [];
+
+let focusedWindowId = undefined;
+let currentWindowId = undefined;
+
+async function bootstrap() {
+  const currentWindow = await chrome.windows.getCurrent();
+  currentWindowId = currentWindow.id;
+  const focusedWindow = await chrome.windows.getLastFocused();
+  focusedWindowId = focusedWindow.id;
+  loadWindowList();
+}
+
+function isInt(i) {
+  return typeof i == 'number' && !(i % 1) && !isNaN(i);
+}
+
+const windowTemplate = document.getElementById('windowItem').content;
+const tabTemplate = document.getElementById('tabItem').content;
+
+async function loadWindowList() {
+  const windowList = await chrome.windows.getAll({ populate: true });
+  tabs = {};
+  tabIds = [];
+  for (let i = 0; i < windowList.length; i++) {
+    windowList[i].current = windowList[i].id == currentWindowId;
+    windowList[i].focused = windowList[i].id == focusedWindowId;
+
+    for (let j = 0; j < windowList[i].tabs.length; j++) {
+      tabIds[tabIds.length] = windowList[i].tabs[j].id;
+      tabs[windowList[i].tabs[j].id] = windowList[i].tabs[j];
+    }
+  }
+
+  const output = document.getElementById('windowList');
+  output.innerHTML = '';
+
+  for (let window of windowList) {
+    const windowItem = document.importNode(windowTemplate, true);
+    renderWindow(window, windowItem.children[0]);
```

Comment:
Do we actually need `windowItem.children[0]` here or would `windowItem` be ok?

Reply:
I think it is necessary to have `windowItem.children[0]` here. 
`windowItem` is a document fragment, and `windowItem.children[0]` refers to the element with the class name "window-item". Modifying it directly as `windowItem` would prevent further modifications such as `windowItem.id`.